### DISCUSSION TITLE
Failed to connect on Synology (both host and bridge) for Non-Crossplay

### DISCUSSION THREAD

**Discussion by IslandJohn** [lloesche/valheim-server-docker#603] [General]
I'm trying to connect to an instance I set up on my Synology DS920+. I have a bridged setup for a Minecraft server, which works fine.

The container seem to get to the fully running state:

- Mar 26 01:09:45 supervisord: valheim-server 03/26/2023 01:09:45: Game server connected

However trying to connect to it on the local network just gets "Failed to connect".

I've tried both host and bridge (with the appropriate 2456/2457 port mapping) networking, with the same result.
I've also tried both SERVER_PUBLIC false (default) and true.

Nothing shows up in the logs during the connection attempt. Should something appear?

If enable -crossplay then it works (using the server list or invite code), but this proxies everything over the Internet.

**Comment by jrwhitehead** [comment]
What are your environment variables set to?
